classes/PaymentModule.php

```diff
@@ -947,6 +948,7 @@ protected function createOrderFromCart(
         $debug,
         $order_status,
         $id_order_state,
+        $note,
```

Comment:
@ks129 You can't add `$note` here, else it's a BC.

Could you add it affter `$carrierId` ?